src/store/actions/authActions.js

```diff
@@ -31,13 +34,23 @@ export const login = (email, password, history) => {
   };
 };
 
-export const logout = (history,expired) => {
+export const logout = (history, expired) => {
   return async (dispatch, getState) => {
-    history.push("/login");
-    dispatch(clearHackDetails())
-    dispatch(logginOut());
-    if(expired){
-      dispatch(SessionExpired())
+    const { auth } = getState();
+    const { jwt } = auth;
```

Comment:
can just do

```
const { auth: { jwt } } = getState();
```

Reply:
never knew how to destructured a nested object, that's cool, thank you!